README.md

```diff
@@ -10,3 +10,31 @@ This is an express endpoint which calls ElasticSearch and returns the results.
 # Search page
 This is a small react app using redux.
 
+# Development
+
+### setup blockbuilder dev environment
+
+Follow the [full dev setup guide](https://github.com/enjalot/blockbuilder/wiki/Development#development) at the **blockbuilder** code-editor project
+
+Once you've done that, verify that it worked by opening a web browser and visiting local blockbuilder at [http://[::]:8889](http://[::]:8889)
+
+### setup search index backend
+
+Follow the steps in the [Setup Elasticsearch & Index some Gists](https://github.com/enjalot/blockbuilder-search-index#setup-elasticsearch--index-some-gists) section of the **blockbuilder-search-index** project
```

Comment:
should we link to the specific elasticsearch version? the readme doesn't say anything about installing elasticsearch
https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-4

Reply:
yes we should.  after this PR https://github.com/enjalot/blockbuilder-search-index/pull/38, the **blockbuilder-search-index**  **README.md** should link to that specific version of Elasticsearch.  

line https://github.com/enjalot/blockbuilder-search-index/pull/38/files#diff-04c6e90faac2675aa89e2176d2eec7d8R120